src/components/standard/checkbox/PECheckboxProps.ts

```diff
@@ -1,6 +1,8 @@
 import { type CheckboxProps } from '@mui/material';
+import React from 'react';
 
 export interface PECheckboxProps extends CheckboxProps {
     disabled?: boolean;
     checked?: boolean;
+    onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
We don't need the entire event back.
```suggestion
    onCheckedChange?: (changedChecked: boolean) => void;
```